java-util/pom.xml

```diff
@@ -107,6 +107,11 @@
             <scope>test</scope>
             <optional>true</optional>
         </dependency>
+      <dependency>
+        <groupId>net.thisptr</groupId>
+        <artifactId>jackson-jq</artifactId>
+        <version>RELEASE</version>
```

Comment:
Please fix this to a specific version, and also, please put the version in the dependencyManagement section of the main pom.